docroot/themes/custom/uids_base/scss/admin.scss

```diff
@@ -88,7 +93,7 @@ $imgpath: '../../uids/assets/images';
     &:hover,
     &:focus {
       .layout-builder__layout {
-        .lb__container {
+        .layout__spacing_container {
```

Comment:
Looks like this scoping places the editor outline on the interior container and not the selected padding. I think the outline can be moved up to the `.layout-builder__layout`, but not sure if that would cause regressions elsewhere.

![image](https://user-images.githubusercontent.com/49074566/93912469-6e74dd00-fcc9-11ea-92d3-92fa38be7282.png)

Reply:
Nice catch, @cory-skeers!